client/src/components/CodeSnippet/CodeSnippet.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC, useEffect } from 'react';
+import { Box, Typography } from '@mui/material';
```

Comment:
FYI: if you're using vscode, shift+alt+o removes unused imports and sorts them for you: https://stackoverflow.com/questions/46722701/is-there-a-way-to-remove-unused-imports-and-declarations-from-angular-2

Reply:
Oh cool. I fixed for CodeSnippet.tsx but how can i remove all unused imports for the whole project easily?

Reply:
Interesting, I can't find such a functionality for VSCode. I think this ends up being a responsibility for the linter: https://stackoverflow.com/questions/64365300/how-can-i-remove-unused-imports-declarations-from-the-entire-project-of-react-ty.